util/topsql/reporter/reporter_test.go

```diff
@@ -250,17 +237,17 @@ func (s *testTopSQLReporter) TestCollectAndTopN(c *C) {
 		}
 		return int(total)
 	}
-	c.Assert(results[0].SqlDigest, IsNil)
-	c.Assert(getTotalCPUTime(results[0]), Equals, 5)
-	c.Assert(results[0].RecordListTimestampSec, DeepEquals, []uint64{0, 1, 3, 4})
-	c.Assert(results[0].RecordListCpuTimeMs, DeepEquals, []uint32{1, 2, 1, 1})
-	c.Assert(results[1].SqlDigest, DeepEquals, []byte("sqlDigest1"))
-	c.Assert(getTotalCPUTime(results[1]), Equals, 5)
-	c.Assert(results[2].SqlDigest, DeepEquals, []byte("sqlDigest3"))
-	c.Assert(getTotalCPUTime(results[2]), Equals, 3)
+	require.Nil(t, results[0].SqlDigest)
+	require.Equal(t, getTotalCPUTime(results[0]), 5)
+	require.Equal(t, results[0].RecordListTimestampSec, []uint64{0, 1, 3, 4})
+	require.Equal(t, results[0].RecordListCpuTimeMs, []uint32{1, 2, 1, 1})
+	require.Equal(t, results[1].SqlDigest, []byte("sqlDigest1"))
+	require.Equal(t, getTotalCPUTime(results[1]), 5)
+	require.Equal(t, results[2].SqlDigest, []byte("sqlDigest3"))
+	require.Equal(t, getTotalCPUTime(results[2]), 3)
```

Comment:
Reverted expect/target for these lines, should be `require.Equal(t, 3, getTotalCPUTime(results[2]))` or so.

Reply:
ok, I will fix it.